lib/sqlitex/server.ex

```diff
@@ -15,6 +15,10 @@ defmodule Sqlitex.Server do
   {:ok, [[a: 2, b: 2]]}
   iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 1", into: %{})
   {:ok, [%{a: 1, b: 1}]}
+  iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 2", into: :raw_list)
+  {:ok, [[1, 1], [2, 2]], [:a, :b]}
+  iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 2", into: :raw_list, types: true)
+  {:ok, [[1, 1], [2, 2]], [:a, :b], [:INTEGER, :INTEGER]}
```

Comment:
OK, that change makes sense. Give me a day or three to work that up. (Back to work tomorrow; not sure if I can get to it before I'm back in the swing of things …)